app/styles/app/layouts/broadcasts.sass

```diff
@@ -49,7 +48,7 @@
     text-decoration: underline
 
   &.is-open
-    z-index: 90
+    @include z-index(above-above)
```

Comment:
I’m going to come back to this, I’m undecided about how this should work with flashes